tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -404,6 +405,13 @@ bool SwiftASTManager::initCompilerInvocation(CompilerInvocation &Invocation,
   ClangImporterOptions &ImporterOpts = Invocation.getClangImporterOptions();
   ImporterOpts.DetailedPreprocessingRecord = true;
 
+  llvm::InitializeAllTargets();
+  llvm::InitializeAllTargetMCs();
+  llvm::InitializeAllAsmPrinters();
+  llvm::InitializeAllAsmParsers();
```

Comment:
What happened here?  Also, why wouldn't this be covered by your change below that only gets called once instead of on every new invocation?